packages/data-stores/src/add-ons/types.ts

```diff
@@ -6,12 +6,11 @@ export interface AddOnMeta {
 	addOnSlug: AddOnSlug;
 	productSlug: StoreProductSlug;
 	featureSlugs?: string[] | null;
-	icon: JSX.Element;
+	icon?: JSX.Element;
```

Comment:
This has become optional since I'd like `AddOnMeta` to be able to represent an add-on in the minimal possible way, and that's `addOnSlug` and `productSlug`.